src/utils/background.py

```diff
@@ -83,8 +83,8 @@ async def old_ticket_actions(cls, bot: commands.Bot, guild: discord.Guild,
         log.info(f"check: {check}- {channel}")
 
         if check == 1:
-            close = actions.CloseTicket(guild, bot, channel, background=True)
-            await close.main(inactivity=True)
+            close = actions.CloseTicket(guild, bot, channel, bot=True)
+            await close.main(before_message="This ticket was automatically closed due to inactivity.")
```

Comment:
is this only send when they don't reply at first, or will it be sent when ac closes it as well? If only the first I think it should be changed to "This ticket was automatically closed due to no response." or "This ticket was automatically closed due to no response from user."

Reply:
Good catch, will update right now.